qcodes/instrument_drivers/tektronix/Keithley_2450.py

```diff
@@ -108,27 +294,59 @@ def __init__(self, parent: 'Keithley2450', name: str, proper_function: str) -> N
             vals=Numbers(0, 1e4)
         )
 
-    def _measure(self) -> str:
+        self.add_parameter(
+            'auto_zero_enabled',
+            get_cmd=f":SENSe:{self._proper_function}:AZERo?",
+            set_cmd=f":SENSe:{self._proper_function}:AZERo {{}}",
+            val_mapping=create_on_off_val_mapping(on_val="1", off_val="0"),
+            docstring="This command enables or disables automatic updates to"
+                      "the internal reference measurements (autozero) of the"
+                      "instrument."
+        )
+
+        self.add_parameter(
+            'count',
+            get_cmd=":SENSe:COUNt?",
+            set_cmd=":SENSe:COUNt {{}}",
```

Comment:
no need for double braces here
```suggestion
            set_cmd=":SENSe:COUNt {}",
```

Reply:
removed extra braces.